longhaul-test/binding.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: dapr.io/v1alpha1
```

Comment:
Have you validated it in dapr-seattle cluster? seems like it is pointing to local kafka

Reply:
Updated